### PR TITLE
Update link in README

### PR DESCRIPTION
**Pull request by LoraArmstrong**
The link to the CodeRefinery exercise was broken, so I fixed it.

### CODE REVIEW

**Review by annepoot — CHANGES_REQUESTED** [review]
Please add a link to the HackMD as well

**Comment by LoraArmstrong** [comment]
@annepoot the HackMD link isn't shared publicly for this workshop, do you have another suggestion?

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 # Centralized workflow exercise
 
 Exercise description: https://coderefinery.org/git-collaborative/centralized/#exercise-part-1-creating-a-pull-request
+HackMD link: https://hackmd.io/12qv4mNTSSykfFaLqpmA2Q
```
Comment (annepoot) [inline README.md]: This link should not be shared!

**Comment by LoraArmstrong** [comment]
That makes 2 of us...here are the changes twice by mistake (maybe a good candidate to squash & merge!)

**Review by annepoot — APPROVED** [review]
Looks good!

**Comment by annepoot** [comment]
> That makes 2 of us...here are the changes twice by mistake (maybe a good candidate to squash & merge!)

I just did a regular merge, but I think squash+merge was not necessary, because it was all in the same branch? Note 100% sure though...

**Pull request merged**